yarn-project/end-to-end/src/e2e_p2p_network.test.ts

```diff
@@ -47,21 +60,32 @@ describe('e2e_p2p_network', () => {
 
     const initialValidators = [EthAddress.fromString(account.address)];
 
-    // Add 1 extra validator if in devnet or NUM_NODES if not.
-    // Each of these will become a validator and sign attestations.
-    const limit = IS_DEV_NET ? 1 : NUM_NODES;
-    for (let i = 0; i < limit; i++) {
-      const account = privateKeyToAccount(`0x${getPrivateKeyFromIndex(i + 1)!.toString('hex')}`);
-      initialValidators.push(EthAddress.fromString(account.address));
-    }
-
-    ({ teardown, config, logger } = await setup(0, { initialValidators, ...options }));
+    ({ teardown, config, logger, deployL1ContractsValues } = await setup(0, { initialValidators, ...options }));
```

Comment:
i wonder if setup even needs initial validators if it will always contain a single one

Reply:
It will not always contain a single, it will just allow anyone if no initial. For this test, think we will also need to alter it when using larger slots. At least I ran into issues when using a walltime and aztec_slot_duration > ethereum_slot_duration.